scripts/data_files/driver_templates/psa_crypto_driver_wrappers.h.jinja

```diff
@@ -2724,6 +2715,7 @@ static inline psa_status_t psa_driver_wrapper_pake_setup(
         case PSA_KEY_LOCATION_LOCAL_STORAGE:
             /* Key is stored in the slot in export representation, so
              * cycle through all known transparent accelerators */
+            status = PSA_ERROR_NOT_SUPPORTED;
```

Comment:
Won't this cause "dead store" warnings in cases where `PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT` and `PSA_CRYPTO_DRIVER_TEST` are defined?

Reply:
That doesn't seem to be something that compilers complain about. I can't test with IAR when I'm not in the office (network license limitations), but certainly I didn't see warnings yesterday, and I don't see warnings with clang or gcc with all warnings enabled, if I write e.g. `x = 1; x = 2;`

Reply:
Actually, after testing, even clang's `scan-build` doesn't complain about this one, so it should be okay